docs/index.html

```diff
@@ -44,6 +44,18 @@ <h2 id="swagger--summary-no-tags">Summary</h2>
         
                         </td>
                     </tr>
+                    <tr>
+                            <td class="swagger--summary-path" rowspan="1">
+                                <a href="#path--product">/product</a>
+                            </td>
+                        <td>
+                            <a href="#operation--product-get">GET</a>
+                        </td>
+                        <td>
+                            <p>Retrieve product and pricing information associated with a premium customer subscription.</p>
```

Comment:
@vickywane this description is misleading. If the method returns information associated with a subscription, then the arguments to the method should be user id or subscription id. Needs clean up.

Reply:
Alright. 

You however requested me to use that description [here](https://github.com/ambianic/ambianic-subscriptions/pull/20#discussion_r651870381).

Reply:
Yes, I thought at the time that we would pass user subscription ID as an argument, but later I realized your intention is to pass product ID as an argument to this `GET /product` method.